ts/components/NavBarLabel.tsx

```diff
@@ -70,10 +71,10 @@ const NavBarLabel: React.FunctionComponent<Props> = (props: Props) => {
           color: options.tintColor === null ? undefined : options.tintColor
         }
       ]}
-      accessibilityLabel={I18n.t("navigation.accessibility", {
+      accessibilityLabel={`${I18n.t("navigation.accessibility", {
```

Comment:
On both iOS and Android, when a tab selector is selected, the status is first read (if selected). For example, opening Facebook (or Whatsapp) and selecting an active tab, is read:
`È selezionato, sezione notizie, pannello, 1 di 5`
What you think about adhering to the standard behaviour and reading the selected state first (if any)?

Reply:
For me is ok, @thisisjp what do you think?

Reply:
sure!

Reply:
Fixed as suggested! ✅


https://user-images.githubusercontent.com/53444973/125268284-d7d0e580-e307-11eb-8326-fb46642bca1a.mp4